client/webserver/site/src/js/order.ts

```diff
@@ -265,5 +265,9 @@ const CoinExplorers: Record<number, Record<number, (cid: string) => string>> = {
   2: { // ltc
     [Mainnet]: (cid: string) => `https://ltc.bitaps.com/${cid.split(':')[0]}`,
     [Testnet]: (cid: string) => `https://tltc.bitaps.com/${cid.split(':')[0]}`
+  },
+  3: { // doge
+    [Mainnet]: (cid: string) => `https://dogechain.info/tx/${cid.split(':')[0]}`,
```

Comment:
I take back my suggestion to use dogechain.info. Official or not, it's not so good.  CloudFlare constantly bugging and it the site itself isn't the best.  I think https://dogeblocks.com/tx/ is better.  No external JS and it actually shows the tech details we care about.  blockchair hurts my eyes but it almost works